### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by samichoulo911**
Merge #20 before.
This adds a logging crate that uses env_variables to determine logging level. The levels are trace-debug-info-warn-error. 
Ex: `RUST_LOG=trace cargo run`

### CODE REVIEW

File: src/config_parser.rs [hunk 1]
```diff
@@ -19,8 +19,8 @@ impl TurboAudioConfig {
         {
             Ok(settings) => settings,
             Err(error) => {
-                println!("{:?}", error);
-                println!("Failed to get settings file. Default values used");
+                log::error!("{:?}", error);
+                log::info!("Failed to get settings file. Default values used");
```
Comment (SGauvin) [inline src/config_parser.rs]: jmettrais un warn ici

**Comment by SGauvin** [comment]
LGTM,
Ya tu des couleurs tho

**Review by SGauvin — DISMISSED** [review]

**Comment by samichoulo911** [comment]
image

File: src/config_parser.rs (hunk 1 above)
Comment (SGauvin) [reply src/config_parser.rs]: @samichoulo911 va chier mon caliss

Comment (SGauvin) [reply src/config_parser.rs]: Oops tas change javais pas spot hahaha

**Pull request merged**